src/amo/components/SearchForm/Suggestion.js

```diff
@@ -0,0 +1,39 @@
+/* @flow */
+/* eslint react/no-array-index-key: 0 */
+import React from 'react';
+import escapeStringRegexp from 'escape-string-regexp';
+
+type Props = {|
+  search: string,
+  suggestion: string,
+|};
+
+const Suggestion = ({ search, suggestion }: Props) => {
+  let parts;
+  if (search.replace(/\s+/, '') === '') {
+    parts = [suggestion];
+  } else {
+    // split the suggestion by the search typed by the user so that it can be
+    // identified as "matched" (.match-search class) below. It needs to be
+    // escaped since template strings are not by default.
+    parts = suggestion.split(new RegExp(`(${escapeStringRegexp(search)})`, 'gi'));
+  }
+
+  return (
+    <p className="Suggestion">
+      {parts.map((text, idx) => {
+        if (text.toLowerCase() === search.toLowerCase()) {
+          return (
+            <span className="match-search" key={`${suggestion}-${text}=${idx}`}>
```

Comment:
We don't have a tonne of typed-up guidelines but [we do have some for CSS](https://github.com/mozilla/addons-frontend/blob/master/docs/css.md). It would be great if you used component names to make `className`s easily searchable/identifiable that'd be great ^_^